src/input_demo/main.cpy

```diff
@@ -2,6 +2,66 @@
 #include "../build/rmkit.h"
 using namespace std
 
+class GestureWidget : public ui::Widget:
+  public:
+  int square_x, square_y
+  int size = 60
+  int drag_x, drag_y
+  remarkable_color default_fill = BLACK
+  remarkable_color fill = BLACK
+  remarkable_color outline = BLACK
+  remarkable_color widget_outline = BLACK
+
+  GestureWidget(int x, y, w, h) : ui::Widget(x, y, w, h):
+    square_x = w / 2
+    square_y = h / 2
+    // drag events
+    gesture.drag_start += PLS_LAMBDA(auto & ev) {
```

Comment:
does `gestures` (plural) reads better than `gesture`? (i don't have much to say about the code itself, apparently ;) )

Reply:
🤷 either way is good with me. I thought about both but couldn't really decide

Reply:
plural, then